torch_agents/cogment_verse_torch_agents/hive_adapter/hive_agent_adapter.py

```diff
@@ -74,6 +75,7 @@ def _load(self, model_id, version_number, version_user_data, model_data_f):
             id=model_id,
             obs_dim=int(version_user_data["obs_dim"]),
             act_dim=int(version_user_data["act_dim"]),
+            qnet=FunctionApproximator(MLPNetwork)(hidden_units=256),
```

Comment:
To go back to the atari thing this is something that could be parametrized by something in the `user_data`